.github/hooks/pre-commit

```diff
@@ -5,6 +5,12 @@
 # ln -s ../../.github/hooks/pre-commit .git/hooks/pre-commit
 #
 
+REACT_FILES=$(git diff --cached --name-only administration/ | sed 's| |\\ |g')
```

Comment:
```suggestion
REACT_FILES=$(git diff --cached --name-only -- administration/ | sed 's| |\\ |g')
```

I guess we also need -- here:

```
fatal: ambiguous argument 'administration/': unknown revision or path not in the working tree.
Use '--' to separate paths from revisions, like this:
'git <command> [<revision>...] -- [<file>...]'
```

Reply:
hm i expected this should work `bash .github/hooks/pre-commit` but doesn't

Reply:
yes it does work, with this command: bash .github/hooks/pre-commit
but it will do nothing if there are not staged files and skip all steps and therefore create no output

Reply:
i think this proposed adjustment has to be done. Rest looks fine to me then

Reply:
I actually meant, how to make it work automatically, as a hook.
For example, I make some changes in administration and click 'commit' in Idea.
As I understand, there must be some output from this hook, but I don't get where to look for it.

Reply:
You need to set you hook path in the git config to the .github/hooks directory. You can do this by
`git config --local core.hooksPath .github/hooks`